agents/langchain-deepagents-code/dcode-session-supervisor.py

```
@@ -0,0 +1,148 @@
+
+
+def _exit_code(returncode: int) -> int:
+    return returncode if returncode >= 0 else 128 + abs(returncode)
+
+
+def run(argv: Sequence[str]) -> int:
+    if not argv:
+        print("dcode session supervisor requires a command.", file=sys.stderr)
+        return 64
+    if sys.platform != "linux":
+        os.execvp(argv[0], list(argv))
```

Ruff (0.15.20)

[error] Starting a process without a shell

(S606)